ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageMessage.qml

```diff
@@ -31,19 +31,24 @@ Rectangle {
         }
 
         Image {
+            // This minimum height is for the initial load of the chat to work correctly
+            // Without this, the chat doesn't understand that the image wil lhave a height and doesn't scroll to the bottom
+            property int minHeight: 400
```

Comment:
How does QML read this custom property in a way that affects scrolling? Is it possible to achieve a similar affect by setting the [`Image.height` property](https://doc.qt.io/qt-5/qml-qtquick-item.html#height-prop) instead?

Reply:
QML doesn't understand `minHeight`, I just use it on line 14 if the image height is 0. I thought of trying `height` directly, but I don't know if doing `height: this.height || minHeight` works. I'll try

Reply:
Just tested, `height: this.height || minHeight` doesn't work